tox.ini

```diff
@@ -0,0 +1,17 @@
+[tox]
+envlist = lint,py36
+
+[testenv]
+setenv =
+    PYCURL_SSL_LIBRARY=openssl
+deps =
+    -rtest-requirements.txt
+commands =
+    python pcs/test/suite.py -v --vanilla
+
+[testenv:lint]
+skip_install = true
+deps =
+    -rtest-requirements.txt
+commands =
+    pylint --rcfile=pylintrc --disable=C,R,W pcs
```

Comment:
I am not sure how `pylint` is actually used. I added this `lint` task as I found `pylintrc` in this repository.